modules/metastorage-api/src/main/java/org/apache/ignite/internal/metastorage/dsl/OperationType.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.ignite.internal.metastorage.common;
+package org.apache.ignite.internal.metastorage.dsl;
```

Comment:
I really don't like the `dsl` name. There is still a difference between a language and an API. Really minor, but I can't be silent :)

Reply:
Sure, do you have a better name in mind?